src/app/home/home.component.ts

```diff
@@ -19,12 +19,11 @@ export class HomeComponent implements OnInit {
         filter((msg: EventMessage) => msg.eventType === EventType.LOGIN_SUCCESS),
       )
       .subscribe((result: EventMessage) => {
-        console.log(result);
         const payload = result.payload as AuthenticationResult;
         this.authService.instance.setActiveAccount(payload.account);
+        this.setLoginDisplay();
       });
-
-      this.setLoginDisplay();
+       this.setLoginDisplay();
```

Comment:
```suggestion
       this.setLoginDisplay();
```
can you check the indentation here?

Reply:
Done.